deluge/tests/twisted/plugins/delugereporter.py

```diff
@@ -5,43 +5,44 @@
 # See LICENSE for more details.
 #
 
```

Comment:
Just delete this file?

Reply:
That was my thought, was just seeing if anyone remembered what it was for first.